table.js

```diff
@@ -0,0 +1,11 @@
+ 
+        
+        const arr = [{"name": "Harsh", "mail":"Hvwaghela@", "gender":"male", "hobbies": ["sports"], "country":"india", "state":"gujarat", "city":"surat"}, 
+        {"name": "Harry", "mail":"Harrywaghela@", "gender":"male", "hobbies": ["sports", "Reading"], "country":"india", "state":"gujarat", "city":"surat"}]
+        window.onload = function(){
+            for(let i of arr){
```

Comment:
Change name `i` to something meaningful.

Reply:
Okay Sir